cpp/src/cider-velox/benchmark/CMakeLists.txt

```diff
@@ -50,11 +50,9 @@ set(FOLLY_WITH_DEPENDENCIES ${FOLLY_LIBRARIES} ${Boost_LIBRARIES}
 set(FOLLY_BENCHMARK Folly::follybenchmark)
 set(FOLLY ${FOLLY_LIBRARIES})
 set(VELOX_BENCHMARKS_DEPENDENCIES
-    cider_processor
-    cider_plan_substrait
+    cider_static
     velox_plan_transformer
     cider_plan_transformer
```

Comment:
can we just export cider_velox static library, remove cider_plan_transformer and others in cider-velox

Reply:
yes, it's the final goal, will provide a single library libvelox_plugin for presto integration and libcider for other developer